proto/src/determined/api/v1/user.proto

```diff
@@ -66,3 +66,34 @@ message PatchUserResponse {
   // The updated user.
   determined.user.v1.User user = 1;
 }
+// Get user settings.
+message GetUserSettingRequest {
+  // The id of the user.
+  int32 user_id = 1;
```

Comment:
if- later on- we wanted to adapt the GET API to return a particular setting as opposed to all of them for a user, would we be able to accommodate that pretty well?